ajax/libs/modern-normalize/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "modern-normalize",
+  "description": "Normalize browsers' default style",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sindresorhus/modern-normalize.git"
+  },
+  "author": "Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)",
+  "filename": "modern-normalize.min.css",
+  "keywords": [
+    "normalize",
+    "css",
+    "reset",
+    "browser",
+    "style"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sindresorhus/modern-normalize.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "modern-normalize.css"
```

Comment:
Let's use `modern-normalize*`
Because this pattern is more flexible to grab the files.
Also, if the repo release `modern-normalize.min.css`, we can host it without resend the PR to modify this pattern.

Many thanks.